apps/meteor/tests/data/livechat/department.ts

```diff
@@ -59,29 +58,19 @@ new Promise((resolve, reject) => {
 
 export const createDepartmentWithAnOnlineAgent = async (): Promise<{department: ILivechatDepartment, agent: {
 	credentials: IUserCredentialsHeader;
-	user: IUser;
+	user: IUser & { username: string };
 }}> => {
-	// TODO moving here for tests
-	const username = `user.test.${Date.now()}`;
-	const email = `${username}@rocket.chat`;
-	const { body } = await request
-			.post(api('users.create'))
-			.set(credentials)
-			.send({ email, name: username, username, password });
-	const agent = body.user;
-	const createdUserCredentials = await login(agent.username, password);
-	await createAgent(agent.username);
-	await makeAgentAvailable(createdUserCredentials);
+    const { user, credentials } = await createAnOnlineAgent();
 
 	const department = await createDepartmentWithMethod() as ILivechatDepartment;
 
-	await addOrRemoveAgentFromDepartment(department._id, {agentId: agent._id, username: (agent.username as string)}, true);
+	await addOrRemoveAgentFromDepartment(department._id, {agentId: user._id, username: (user.username as string)}, true);
```

Comment:
```suggestion
	await addOrRemoveAgentFromDepartment(department._id, {agentId: user._id, username: user.username }, true);
```

If you're adding `username` as non optional, you shouldn't need this